### ISSUE TITLE
Inaccurate locked value

### ISSUE THREAD

**Issue by Falci** [kyokan/bob-wallet#405]
**TL;DR:** name claim reward is not appearing in the "Locked" balance card.

My wallet show some balance as **Spendable** and 0 as **Locked**
In another computer, I have an old version of Bob, from before the balance cards, with the "Total Balance" and "Unlocked Balance" using the same wallet/seed.
The old Bob shows different values, meaning I have some balance locked.
The value correspond to a name claim.

## Expected Behavior
Show the name claim reward as locked balance

## Current Behavior
Locked balance card doesn't include the name claim reward

## Steps to Reproduce
1. Check your Spendable and Locked balance
1. Make a name claim (`sendclaim`)
1. Check the Locked balance again
